### PR TITLE
Add new article on resizing compute & memory servers: INFODEV-2314

### PR DESCRIPTION
**Pull request by RackerKate**
Replaces #2734 .

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-63b593e34c/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate On it!

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[resize-up-compute-and-memory-servers_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2337428/resize-up-compute-and-memory-servers_cyrEdits.docx)

File: content/cloud-servers/resize-up-compute-and-memory-servers.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+last_modified_date: '2018-08-30'
+last_modified_by: Kate Dougherty
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+**Previous section:** [Resize standard and general purpose servers](/how-to/resize-standard-and-general-purpose-servers/)
+
+This article shows you how to resize up a computer or memory server by using
+the Cloud Control Panel. Resizing a server changes the RAM and disk space
```
Comment (chri2547) [inline content/cloud-servers/resize-up-compute-and-memory-servers.md]: Maybe "This article shows you how to use the Cloud Control Panel to resize up a computer or memory server."

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-672fd01ba8/how-to/).

**Pull request merged**